### PR TITLE
links to docs like pytorch

### PR DESCRIPTION
**Pull request by ydcjeff**

### CODE REVIEW

**Comment by ydcjeff** [comment]
What is the another one to link?

File: layouts/partials/docs.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      <p>Get in-depth tutorials for beginners and advanced developers</p>
+      <a href="/tutorials/">View Tutorials →</a>
+    </div>
+    <div class="admonition">
+      <h3 class="title-2">How-to-Guides</h3>
+      <p>Get byte-size ready how-to short tips</p>
+      <a href="/how-to-guides/">View How-to-Guides →</a>
+    </div>
+    <div class="admonition">
+      <h3 class="title-2">Another one is what?</h3>
```
Comment (vfdev-5) [inline layouts/partials/docs.html:19]: Developer Guide = Contributing.md or Community page ?

**Comment by vfdev-5** [comment]
Can we make this block with another background color as pytorch does ?

File: layouts/partials/docs.html (hunk 1 above)
Comment (ydcjeff) [reply layouts/partials/docs.html:19]: Then I think having 3 items is enough here as there is a proper contributing.md in ignite repo and community page is not that important to visit on the first visit to the website.

Comment (vfdev-5) [reply layouts/partials/docs.html:19]: OK, let's keep 3 of them

**Comment by vfdev-5** [comment]
@ydcjeff can we have this block between the block on "Simple Engine and Event System" and "Ecosystem" ?

**Review by vfdev-5 — APPROVED** [review]
When the black block goes from the left end to the right end, it doesn't look neat. But ok for now

**Pull request merged**